pkg/pbq/store/options.go

```diff
@@ -0,0 +1,90 @@
+package store
+
+import dfv1 "github.com/numaproj/numaflow/pkg/apis/numaflow/v1alpha1"
+
+type Options struct {
+	// maxBufferSize max size of batch before it's flushed to store
+	maxBatchSize int64
+	// syncDuration timeout to sync to store
+	syncDuration int64
+	// pbqStoreType store type (memory or s3 or file system)
+	pbqStoreType string
+	// ChannelSize buffered channel size
+	bufferSize int64
+	// storeSize store array size
+	storeSize int64
+	// readTimeout timeout in seconds for pbq reads
+	readTimeout int
+}
+
+func (o *Options) StoreSize() int64 {
+	return o.storeSize
+}
+
+func (o *Options) BufferSize() int64 {
+	return o.bufferSize
+}
+
+func (o *Options) ReadTimeout() int {
```

Comment:
```suggestion
func (o *Options) ReadTimeoutSecs() int {
```
if we suffix with `Secs` people will know the timeout is in seconds. also please document all options.

Reply:
done